inventory/group_vars/vagrant

```diff
@@ -27,12 +27,15 @@ nodepool_labels:
 nodepool_providers:
   - name: cicloud
     cloud: cicloud
-    max-servers: 10
+    max-servers: 3
```

Comment:
for developers, we should probably keep this to 1 by default just to ease resource usage on the cloud